lib/eval.coffee

```diff
@@ -34,6 +34,31 @@ module.exports =
           @showError r, result.highlights
         notifications.show "Evaluation Finished"
 
+  # get documentation or methods for the current word
+  toggleMeta: (type) ->
+    editor = atom.workspace.getActiveTextEditor()
+    [word, range] = @getWord editor
+    # if we only find numbers or nothing, return prematurely
+    if word.length == 0 || !isNaN(word) then return
+    client.msg type, {code: word}, ({result}) =>
+      view = if result.type then result.view else result
+      view = @ink.tree.fromJson(view)[0]
+      @ink.links.linkify view
+      r = @ink?.results.toggleUnderline editor, range,
+        content: view
+        clas: 'julia'
+
+  # gets the word and its range in the `editor` which the last cursor is on
+  getWord: (editor) ->
+    cursor = editor.getLastCursor()
+    # The following line is kinda iffy: The regex may or may not be well chosen
+    # and it duplicates the efforts from atom-language-julia. It might be
+    # better to select the current word via finding the smallest <span> containing
+    # the cursor which also has `function` or `macro` as its class.
+    range = cursor.getCurrentWordBufferRange({wordRegex: /[\w_!´]*\.?@?[\w_!´]+/})
```

Comment:
Hm, the regex I posted above seems quite nice...
![regexing](https://cloud.githubusercontent.com/assets/6735977/12025917/c68a2866-adb3-11e5-8a84-d0dbf7ecfce2.PNG)

I'm sure it has loads of problems, but it does seem better than anything else I could come up with. Extracting valid identifiers from Julia's parser is a bit above me, tbh.